tests/integration_tests/capture_local_fence_on_session_done/run.sh

```
@@ -0,0 +1,170 @@
+	check_logs_contains $WORK_DIR "dispatcher orchestrator local fence triggered" 2
+	check_logs_contains $WORK_DIR "stopping dispatcher manager write path" 2
+	check_logs_contains $WORK_DIR "server closed" 2
+
+	run_sql "INSERT INTO ${DB_NAME}.${TABLE_NAME} VALUES (5, 50), (6, 60);" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
+	check_sync_diff $WORK_DIR $CUR/conf/diff_config.toml
+
+	cleanup_process $CDC_BINARY
+}
+
+trap 'stop_test $WORK_DIR' EXIT
+run $*
```

Shellcheck (0.11.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)